secure_fw/partitions/internal_trusted_storage/tfm_its_req_mngr.c

```diff
@@ -32,6 +33,26 @@ static uint8_t __ALIGNED(4) asset_data[ITS_UTILS_ALIGN(ITS_BUF_SIZE,
                                           ITS_FLASH_MAX_ALIGNMENT)];
 #endif
 
+
+static psa_status_t buffer_size_check(int32_t client_id, size_t buffer_size) {
```

Comment:
Hi @mswarowsky .
I've noticed that when building samples for `nrf5340dk_nrf5340_cpuapp_ns` for openthread cli I'm getting a warning: 
```
tfm_its_req_mngr.c:37:21: warning: 'buffer_size_check' defined but not used [-Wunused-function]
   37 | static psa_status_t buffer_size_check(int32_t client_id, size_t buffer_size) {
      |                     ^~~~~~~~~~~~~~~~~
```
I think adding an `#ifdef TFM_ITS_ENCRYPTED` guard here is necessary too.

Reply:
@maciejbaczmanski Already created a PR here: https://github.com/nrfconnect/sdk-trusted-firmware-m/pull/88

Reply:
Oh, didn't see that. Thanks!